.test/docker-compose.yml

```
@@ -0,0 +1,18 @@
+version: '2'
+services:
+  db:
+    container_name: emissionsapi-postgres
+    image: postgis/postgis:12-master
```

Maybe use the full location:
```suggestion
    image: docker.io/library/postgis/postgis:12-master
```

---

Done. See the `docker-compose.yml` and the `emissionsapi.dockerfile`